HomeWork 5/LoggerAplication/LoggerAplication/Action.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace LoggerAplication
+{
+    internal class Action
```

Comment:
I think this must ne ActionService

Reply:
i didn`t know about Service))) at list this day)